default/AI/AIstate.py

```diff
@@ -120,8 +127,28 @@ def __setstate__(self, state_dict):
         for odict_attrib in ['colonisablePlanetIDs', 'colonisableOutpostIDs']:
             if dict_attrib not in state_dict:
                 self.__dict__[odict_attrib] = odict()
+        if 'uuid' not in state_dict:
+            self.uuid = uuid4()
+        if 'turn_hashes' not in state_dict:
+            self.turn_uuids = {}
         self.__dict__.setdefault('empire_standard_enemy_rating', 40)
 
+    def set_turn_uuid(self):
+        """
+        Set turn uuid. Should be called once per generateOrders
```

Comment:
Lets consider that `uuid` is name of file where dump saved. 
How many files you get when you load game for 5 times make turn and exit?